kolibri/utils/conf.py

```diff
@@ -44,7 +45,8 @@
 # Create a folder named logs inside KOLIBRI_HOME to store all the log files.
 LOG_ROOT = os.path.join(KOLIBRI_HOME, "logs")
 if not os.path.exists(LOG_ROOT):
-    os.mkdir(LOG_ROOT)
+    if not (NO_FILE_BASED_LOGGING and str(NO_FILE_BASED_LOGGING).lower() == "true"):
```

Comment:
Sure. I just added the extra condition just in case if the user sets the environment variable to false explicitly. In that case, without this condition, the "logs" folder would still be created. But if we don't need to consider that, I'll remove the extra conditionals.